olmeiro-orozco/tarea-2/tarea2.1-Cover/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Cover Magazine</title>
+  </head>
+
+  <body>
+    <main>
+      <header>
```

Comment:
@OlmeiroOrozco there should be an article enclosing all the sections

Reply:
ok, I will